zerver/models.py

```diff
@@ -391,6 +391,10 @@ class UserProfile(ModelReprMixin, AbstractBaseUser, PermissionsMixin):
     last_pointer_updater = models.CharField(max_length=64) # type: text_type
     realm = models.ForeignKey(Realm) # type: Realm
     api_key = models.CharField(max_length=32) # type: text_type
+    tos_version = models.CharField(default='0.0', max_length=10) # type: text_type
+    # For the Aug 2016 migration from zulip.com to zulipchat.com, that
+    # required everyone to sign a new TOS
+    ZULIPCHAT_MIGRATION_TOS_VERSION = u'0.0-zc'
```

Comment:
OK, so I think we should make the default here `settings.TOS_VERSION`, the default in zproject/settings.py be None (And make this a nullable field), and we'll just plan to set `settings.TOS_VERSION` to something like `1.0` on the servers we're migrating to.